packages/dx-react-chart-material-ui/src/templates/legend/label.test.jsx

```diff
@@ -1,22 +1,22 @@
 import * as React from 'react';
-import { createShallow, getClasses } from '@material-ui/core/test-utils';
+import { createMount, getClasses } from '@material-ui/core/test-utils';
 import ListItemText from '@material-ui/core/ListItemText';
 import { Label } from './label';
 
 const defaultProps = { text: 'a' };
 
 describe('Label', () => {
-  const shallow = createShallow({ dive: true });
+  const mount = createMount({ dive: true });
```

Comment:
I'm not sure `{ dive: true }` is a required option for `mount`.

Reply:
I'll check it.

Reply:
Yes, there is no `dive` option for `createMount`.
`createShallow({ dive: true })` cannot be used as it gives [one](https://material-ui.com/guides/testing/#api) level of depth.
Redundant option has been removed.